Maths/PrimeCheck.js

```diff
@@ -11,8 +11,9 @@ const PrimeCheck = (n) => {
   // output: boolean
   if (n === 1) return false
   if (n === 0) return false
+  if (n % 2 === 0) return false

```

Comment:
This will lead to 
```js
> PrimeCheck(2)
false
```

Reply:
I somehow deleted the fork. Please see #596 for the update.